Tools/Cli-LoRa-Device-Provisioning/Tests/LoRaWan.Tools.CLI.Tests.Unit/DeviceProvisioningTest.cs

```diff
@@ -17,20 +17,24 @@ public class DeviceProvisioningTest
         private readonly Mock<RegistryManager> registryManager;
 
         private const string NetworkName = "quickstartnetwork";
-        private const string DevEUI = "46AAC86800430028";
+        private readonly string devEUI = GetRandomHexNumber(16);
```

Comment:
Do we want different values per test, or same value for all tests?
If we want same value for all tests we could make this static (as `private static readonly`)